Cargo.toml

```diff
@@ -62,7 +62,7 @@ cluster = ["crc16", "rand"]
 script = ["sha1"]
 async-std-comp = ["aio", "async-std"]
 tokio-comp = ["aio", "tokio"]
-connection-manager = ["tokio-rt-core", "arc-swap", "futures"]
+connection-manager = ["futures", "tokio-rt-core", "arc-swap", "futures"]
```

Comment:
Is this duplicate value somehow necessary?  Curious as I'm working on a merge into #319.  I think this can be trimmed.  

Thanks!

Reply:
Yeah, that can probably be removed.

Reply:
Cool, I've updated the #319.  Cheers.